lib/nopt.js

```diff
@@ -12,9 +12,9 @@ exports.clean = clean
 exports.typeDefs = defaultTypeDefs
 exports.lib = lib
 
-function nopt (types = {}, shorthands = {}, args = process.argv, slice = 2) {
+function nopt (types, shorthands, args = process.argv, slice = 2) {
   return lib.nopt(args.slice(slice), {
-    types,
+    types: types || {},
```

Comment:
There is an existing test where `types` is explicitly set to `null`. Previously this was guarded inside lib, but I made the decision to do less guards in there, so this fallback was the only place it was needed. Getting to 100 coverage helped here since I could make sure each branch was called.